drizzlepac/util.py

```diff
@@ -436,25 +436,31 @@ def updateNEXTENDKw(fobj):
     if 'nextend' in fobj[0].header:
         fobj[0].header['nextend'] = len(fobj) - 1
 
-def count_sci_extensions(filename):
+def count_sci_extensions(filename, return_ind=False):
     """ Return the number of SCI extensions and the EXTNAME from a input MEF file.
```

Comment:
Add documentation that explains "if _return_ind=True_, only a list of the true "SCI" extension indices in the MEF will be returned."  

This is in case someone tries to use this function on files which have been manipulated.  The Primary Data Unit (PDU) of a FITS file can hold images (i.e., tables or such cannot be in the Primary), but this is not the convention used for HST data.

Reply:
Added!